### ISSUE TITLE
Bad rows, regex fails to parse user-agent.

### ISSUE THREAD

**Issue by pkallos** [snowplow/snowplow#576]
I am seeing about 5% of my records marked as bad because the user agent parsing fails. Example:

```json
{
    "line": {
        "timestamp": 1395299873568,
        "collector": "ssc-0.1.0-kinesis",
        ...
        "headers": [..., "User-Agent: Mozilla/5.0 (Windows NT 6.1; Trident/7.0; rv:11.0) like Gecko", ...],
        "networkUserId": "7e98e0a0-e316-40ae-94f4-5a723264397c",
		....
        "setNetworkUserId": true
    },
    "errors": ["Exception parsing useragent [Mozilla/5.0 (Windows NT 6.1; Trident/7.0; rv:11.0) like Gecko]: [No group 1]"]
}
```

**Comment by fblundun** [comment]
Can I close this? I think it's covered by #62 and #792.

**Comment by alexanderdean** [comment]
Plus the underlying bug was fixed in user-agent utils 1.12, which @pkallos pulled in via #662. So yes let's close.

**Comment by alexanderdean** [comment]
Cleared milestone too
